patient-application/src/app/feedback.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { Content } from '@angular/compiler/src/render3/r3_ast';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class RandomNumberGeneratorService {
+
+  private _url = "http://localhost:16934/api/"
```

Comment:
Hardcoded URL, might be good to define them all in a separate file for easier later fixes

Reply:
Refactored